iothub/device/src/InternalClient.cs

```diff
@@ -1841,12 +1752,9 @@ internal async Task OnModuleEventMessageReceivedAsync(string input, Message mess
                     _moduleReceiveMessageSemaphore.Release();
                 }
 
-                // codes_SRS_DEVICECLIENT_33_002: [ The OnReceiveEventMessageCalled shall invoke the specified delegate. ]
```

Comment:
Thanks for removing these comments. It's been far too long since they were maintained, and they are unnecessarily crowding up our files.